x-pack/test/fleet_api_integration/apis/settings/update.ts

```diff
@@ -18,7 +18,8 @@ export default function (providerContext: FtrProviderContext) {
   const esClient = getService('es');
   const esArchiver = getService('esArchiver');
 
-  describe('Settings - update', async function () {
+  // Skipped as the Fleet Server hosts settings values are no longer used as of https://github.com/elastic/kibana/issues/137785
```

Comment:
I think this is the right move to skip these tests - these are leftover from when we were using the `/settings` API to create new Fleet Server hosts. AFAIK this is no longer supported, as any Fleet Server Hosts within the Fleet settings saved object will be ignored. @nchaulet please correct me if I'm wrong here.

We'll probably want to remove this API altogether as part of the GA work.

Reply:
Yes we probably want to remove these tests, and this API (at least the `fleet_server_hosts`) part should be removed as part of the GA work (we should already mark them as deprecated in the open API spec and point to new fleet server hosts endpoints)